src/features/todos.ts

```diff
@@ -1,8 +1,21 @@
-import { createSlice } from '@reduxjs/toolkit';
+import { createSelector, createSlice } from '@reduxjs/toolkit';
 import { Todo } from '../types/Todo';
+import { RootState } from '../app/store';
 
 export const todosSlice = createSlice({
   name: 'todos',
   initialState: [] as Todo[],
-  reducers: {},
+  reducers: {
+    set: (_, action) => {
+      return action.payload;
+    },
```

Comment:
The reducer function 'set' is directly returning the action payload, which is fine if the payload is always the complete new state. Ensure that this is the intended behavior, as it will replace the entire state with the payload.